config/webhook/kustomizeconfig.yaml

```diff
@@ -0,0 +1,18 @@
+# the following config is for teaching kustomize where to look at when substituting vars.
+# It requires kustomize v2.1.0 or newer to work properly.
```

Comment:
We require v3.1.0 or newer

Reply:
I created a sample project using kubebuilder 2.0 and copied the file from there